server/views/pages/home-new.html

```diff
@@ -30,6 +30,13 @@
     id: 'recentlyAdded',
     data: recentlyAddedContent
   }) }}
+
+  {{ hubContentBlock({
+    title: 'Explore to Hub',
```

Comment:
Typo? Think this should be "Explore the Hub"

Reply:
updated.